cypress/tsconfig.json

```diff
@@ -1,9 +1,17 @@
 {
   "compilerOptions": {
+    "esModuleInterop": true,
+    "incremental": true,
+    "lib": ["DOM", "DOM.Iterable", "ESNext"],
+    "noEmit": true,
     "strict": true,
-    "target": "es5",
-    "lib": ["es5", "dom"],
-    "types": ["cypress"]
+    "target": "ES5",
+    "tsBuildInfoFile": "../node_modules/.cache/tsbuildinfo/cypress",
```

Comment:
I think the PR description is a bit misleading. This is not "extending" TSConfig from the Cypress config. It would be great if you documented the use case of `tsBuildInfoFile` in regards to that configuration.